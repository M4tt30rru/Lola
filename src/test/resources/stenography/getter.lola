##Import "std.lola"
##Find ##Any(_prot) -> ##Any(_mods) ##Type(_type) ##Identifier(_name);
 ##run {
_getter = 'get' + _name.name[0].upper() + _name.name[1:]
 }
 ##replace ##(_mods) ##(_type) ##(_name); ##(_prot) ##(_type) ##(_getter) () {return this.##(_name);}

protected -> private Integer i;
public -> protected transient String str;
