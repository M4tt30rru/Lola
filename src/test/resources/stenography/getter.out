protected transient String str;
private Integer i;
protected Integer getI() {
 return this.i;
}
public String getStr() {
 return this.str;
}
